crypto/keys/secp256k1/secp256k1.go

```diff
@@ -206,3 +267,56 @@ func (pubKey PubKey) MarshalAminoJSON() ([]byte, error) {
 func (pubKey *PubKey) UnmarshalAminoJSON(bz []byte) error {
 	return pubKey.UnmarshalAmino(bz)
 }
+
+// Address returns an ethereum style addresses
+func (pubKey *PubKeyOld) Address() crypto.Address {
+	return pubKey.Address()
+}
+
+// Bytes returns the pubkey byte format.
+func (pubKey *PubKeyOld) Bytes() []byte {
+	return pubKey.Key
+}
+
+func (pubKey *PubKeyOld) String() string {
+	return fmt.Sprintf("PubKeySecp256k1{%X}", pubKey.Key)
+}
+
+func (pubKey *PubKeyOld) Type() string {
+	return keyType
+}
+
+func (pubKey *PubKeyOld) Equals(other cryptotypes.PubKey) bool {
+	return pubKey.Type() == other.Type() && bytes.Equal(pubKey.Bytes(), other.Bytes())
+}
+
+func (pubKey *PubKeyOld) VerifySignature(msg []byte, sigStr []byte) bool {
+	return pubKey.VerifySignature(msg, sigStr)
```

Comment:
I have the impression this method is calling itself

Reply:
True, forgot to push this 🤦 
[Here](https://github.com/0xPolygon/cosmos-sdk/pull/8/commits/722da0aa266e7f8b721b2556ebd5ec689027ace5) it is, thanks